apps/documentation/src/routes/documentation/cli.mdx

```diff
@@ -1,15 +1,14 @@
-import { Head } from 'tuono'
+import MetaTags from '../../components/meta-tags'
 
-<Head>
-  <title>Tuono - CLI</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/cli" />
-</Head>
+<MetaTags
+  title="Tuono - CLI"
+  canonical="https://tuono.dev/documentation/cli"
+  description="Tuono is the CLI that provides all the needed commands to handle the full-stack project."
+/>
 
 import Breadcrumbs, { Element } from '../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="CLI" />
-</Breadcrumbs>
+<Breadcrumbs breadcrumbs={[{ label: 'CLI' }]} />
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a word (ENGLISH_WORD_REPEAT_RULE)
Suggestions: `Breadcrumbs`
Rule: https://community.languagetool.org/rule/show/ENGLISH_WORD_REPEAT_RULE?lang=en-US
Category: MISC
<!-- __reviewdog__:ChA4YzgzMGJjMmFiZDg5ZTk3EgxMYW5ndWFnZVRvb2w= -->